internal/testhelpers/e2e/session.go

```diff
@@ -165,9 +165,9 @@ func executablePaths(t *testing.T) (string, string, string, string) {
 	if !fileutils.FileExists(trayExec) {
 		t.Fatal("E2E tests require a state-tray binary. Run `state run build-tray`.")
 	}
-	if !fileutils.FileExists(executorExec) {
-		t.Fatal("E2E tests require a state-exec binary. Run `state run build-exec`.")
-	}
+	// if !fileutils.FileExists(executorExec) {
```

Comment:
Does this still need to be disabled?

Reply:
Nope, that was for local testing. Thanks for catching!